hedera-node/src/main/java/com/hedera/services/keys/HederaKeyActivation.java

```diff
@@ -73,7 +74,7 @@ public static boolean payerSigIsActive(
 			final TxnAccessor accessor,
```

Comment:
Can we change this signature to accept a `PlatformTxnAccessor`, since it needs the signature metadata?